nexus/tests/integration_tests/unauthorized.rs

```diff
@@ -152,69 +182,87 @@ lazy_static! {
         },
         // Create an IP pool
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_IP_POOLS_URL,
             body: serde_json::to_value(&*DEMO_IP_POOL_CREATE).unwrap(),
+            id_routes: vec!["/by-id/ip-pools/{id}"],
         },
         // Create an IP Pool range
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_IP_POOL_RANGES_ADD_URL,
             body: serde_json::to_value(&*DEMO_IP_POOL_RANGE).unwrap(),
+            id_routes: vec![],
         },
         // Create an Organization
-        SetupReq {
+        SetupReq::Post {
             url: "/organizations",
-            body: serde_json::to_value(&*DEMO_ORG_CREATE).unwrap()
+            body: serde_json::to_value(&*DEMO_ORG_CREATE).unwrap(),
+            id_routes: vec!["/by-id/organizations/{id}"],
         },
         // Create a Project in the Organization
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_ORG_PROJECTS_URL,
             body: serde_json::to_value(&*DEMO_PROJECT_CREATE).unwrap(),
+            id_routes: vec!["/by-id/projects/{id}"],
         },
         // Create a VPC in the Project
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_PROJECT_URL_VPCS,
             body: serde_json::to_value(&*DEMO_VPC_CREATE).unwrap(),
+            id_routes: vec!["/by-id/vpcs/{id}"],
         },
         // Create a VPC Subnet in the Vpc
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_VPC_URL_SUBNETS,
             body: serde_json::to_value(&*DEMO_VPC_SUBNET_CREATE).unwrap(),
+            id_routes: vec!["/by-id/vpc-subnets/{id}"],
         },
         // Create a VPC Router in the Vpc
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_VPC_URL_ROUTERS,
             body: serde_json::to_value(&*DEMO_VPC_ROUTER_CREATE).unwrap(),
+            id_routes: vec!["/by-id/vpc-routers/{id}"],
         },
         // Create a VPC Router in the Vpc
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_VPC_ROUTER_URL_ROUTES,
             body: serde_json::to_value(&*DEMO_ROUTER_ROUTE_CREATE).unwrap(),
+            id_routes: vec!["/by-id/vpc-router-routes/{id}"],
         },
         // Create a Disk in the Project
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_PROJECT_URL_DISKS,
             body: serde_json::to_value(&*DEMO_DISK_CREATE).unwrap(),
+            id_routes: vec!["/by-id/disks/{id}"],
         },
         // Create an Instance in the Project
-        SetupReq {
+        SetupReq::Post {
             url: &*DEMO_PROJECT_URL_INSTANCES,
             body: serde_json::to_value(&*DEMO_INSTANCE_CREATE).unwrap(),
+            id_routes: vec!["/by-id/instances/{id}"],
+        },
+        // Lookup the previously created NIC
+        SetupReq::Get {
+            url: &*DEMO_INSTANCE_NIC_URL,
+            id_routes: vec!["/by-id/network-interfaces/{id}"],
         },
```

Comment:
This is the (first) usecase of somewhere we may lookup a resource instead of creating a new resource mainly for the purpose of populating the `id` param.